aggregator/src/aggregator.rs

```diff
@@ -2214,6 +2243,40 @@ impl VdafOps {
                                 )
                             )?;
 
+                            // To ensure that concurrent aggregations don't write into a
+                            // currently-nonexistent batch aggregation, we write (empty) batch
+                            // aggregations for any that have not already been written to storage.
+                            let existing_batch_aggregations: HashSet<_> = batch_aggregations
+                                .iter()
+                                .map(|ba| (ba.batch_identifier(), ba.ord()))
+                                .collect();
+                            let empty_batch_aggregations: Vec<_> = iproduct!(
```

Comment:
I agree, but I think this is beyond the scope of this PR. This was already a problem: a too-small time precision would cause a collect request of a few hours to attempt to read many thousands of rows.